### PR TITLE
feat: synchronize with API Platform's definition

### PR DESCRIPTION
**Pull request by dunglas**
* Prevent to use default values for sensitive env vars in production
* Don't store the content of sensitive env vars in the config
* Use a UNIX socket for performance
* Synchronize other changes recently introduced in API Platform

### CODE REVIEW

File: docker/caddy/Caddyfile [hunk 1]
```diff
@@ -14,21 +17,21 @@ route {
...
         # Enable the subscription API (double-check that it's what you want)
         subscriptions
+        # Extra directives
+        {$MERCURE_EXTRA_DIRECTIVES}
     }
     vulcain
     push
-    php_fastcgi php:9000
-    encode gzip
+    php_fastcgi unix//var/run/php/php-fpm.sock
```
Comment (jdreesen) [inline docker/caddy/Caddyfile:34]:
```suggestion
    php_fastcgi unix:/var/run/php/php-fpm.sock
```

Comment (dunglas) [reply docker/caddy/Caddyfile:34]: No, it's the format expected by Caddy: https://caddyserver.com/docs/caddyfile/directives/php_fastcgi

Comment (jdreesen) [reply docker/caddy/Caddyfile:34]: Oh, really. I've never seen it like this. Well, then...

**Review by maxhelias — APPROVED** [review]
Nice improvement 👍

**Pull request merged**